script/cmds.go

```diff
@@ -577,23 +583,22 @@ func Exists() Cmd {
 	return Command(
 		CmdUsage{
 			Summary: "check that files exist",
-			Args:    "[-readonly] [-exec] file...",
+			Args:    "file...",
+			Flags: func(fs *pflag.FlagSet) {
+				fs.Bool("readonly", false, "File must not be writable")
+				fs.Bool("exec", false, "File must not be executable")
```

Comment:
nit: also these flags too, maybe? I was thinking to have a way to be able to re-use these flags that have high chance to be re-used across cmds.